L1Trigger/TrackFindingTMTT/src/L1ChiSquared.cc

```diff
@@ -0,0 +1,183 @@
+///=== This is the base class for the linearised chi-squared track fit algorithms.
+
+///=== Written by: Sioni Summers and Alexander D. Morton
+
+#include "L1Trigger/TrackFindingTMTT/interface/L1ChiSquared.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Matrix.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Stub.h"
+#include "L1Trigger/TrackFindingTMTT/interface/L1fittedTrack.h"
+#include "L1Trigger/TrackFindingTMTT/interface/L1track3D.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Utility.h"
+
+#include <algorithm>
+#include <functional>
+#include <set>
+
+namespace TMTT {
+
+  template <typename T>
+  std::vector<T> operator-(const std::vector<T>& a, const std::vector<T>& b) {
```

Comment:
In general operators should be overloaded in the namespace of the objects. But I would suggest to define a function instead of overloading a subtraction operator. In any case the definition should be moved to anonymous namespace to avoid polluting the `TTMT` namespace at the link time.

Reply:
The subtraction operator was no longer needed, so has been deleted.